libs/pika/async_mpi/src/mpi_polling.cpp

```diff
@@ -622,6 +693,32 @@ namespace pika::mpi::experimental {
 
         int comm_world_size() { return detail::mpi_data_.size_; }
 
+#ifdef OMPI_HAVE_MPI_EXT_CONTINUE
+        void register_mpix_continuation(
+            MPI_Request* request, MPIX_Continue_cb_function* cb_func, void* op_state)
+        {
+            // @TODO we don't strictly need defer complete flag, but for debugging ...
```

Comment:
`MPIX_CONT_DEFER_COMPLETE` is already disabled below, remove comment (and commented out flag)?